### DISCUSSION TITLE
tailwindcss installation (for custom color) not compiling hover button color

### DISCUSSION THREAD

**Discussion by Soif2Sang** [saadeghi/daisyui#2114] [Q&A]
Hello, 

I installed tailwindcss to my project using npm to add custom colors to my project, once I added these colors in the extend section the hover color of the buttons are no longer working (transparent), I tried to implement my own hover color using classes but the transparent colors is showed for 1sec when unhovering the button.

how to compile the daisyui colors ? If not possible, how to change the unhovering color ? Thanks a lot !

**Comment by saadeghi** [comment]
It's not easy to guess what is the problem when you don't provide any code example.  
Please share a minimal project on GitHub or StackBlitz or CodeSandbox
